packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -386,6 +392,11 @@ export class Viewer {
     return this.interactionHandlers;
   }
 
+  @Method()
+  public async getJwt(): Promise<string | undefined> {
+    return this.jwt;
+  }
```

Comment:
is this jwt specifically for STS? might be helpful to specify that in the var/method name.

Reply:
It is right now, but that might not always be the case.